src/commands/txCommands/push.js

```diff
@@ -5,30 +5,13 @@ const txlib = require('./util');
 
 const tx = txlib.tx;
 
+const { pushTxMaster } = require('./pushTxMaster');
+const { pushTxAllTranslations } = require('./pushTxAllTranslations');
+const { gitBranch$, branchCheck$ } = require('./util/gitHelpers');
+
 const readParseResource$ = slug =>
 	txlib.readResource$(slug).flatMap(txlib.parsePluckTrns);
 
-const devGitBranch$ = Rx.Observable.bindNodeCallback(child_process.exec)(
-	'git rev-parse --abbrev-ref HEAD'
-)
-	.pluck(0)
-	.map(str => str.slice(0, -1));
-
-// Branch whether in dev or CI. Replaces forward slashes because
-// branch names are used as transifex resource names and resource
-// names need to work as valid url paths
-const gitBranch$ = Rx.Observable
-	.if(
-		() => process.env.TRAVIS_PULL_REQUEST_BRANCH,
-		Rx.Observable.of(process.env.TRAVIS_PULL_REQUEST_BRANCH),
-		devGitBranch$
-	)
-	.map(branchname => branchname.replace(/\//g, '_'));
-
-const branchResourceExists$ = Rx.Observable
-	.zip(txlib.resources$, gitBranch$)
-	.map(([resources, branch]) => resources.indexOf(branch) > -1);
-
 // syncs content in po format to tx
 const pushResource$ = poData =>
 	Rx.Observable
```

Comment:
in the line below this it's referencing functions that have been to txlib, probably need to update